scripts/run-pr.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
```

Comment:
Hmm, then you can add ` || true` to the npm test line. That way it won't try to checkout a failed fetch, or install a failed checkout. Not exactly the prettiest thing, but at least we'd get at least a bit of error management.